plugins/techdocs-module-addons-contrib/src/LigthBox/lightbox.css

```diff
@@ -5,3 +5,7 @@
   color: rgba(0, 0, 0, 0.87);
   text-shadow: none;
 }
+
+.pswp__img {
+    cursor: pointer !important;
```

Comment:
This is how the library recommends to do it in their documentation https://photoswipe.com/click-and-tap-actions/